src/Sql/Sql/ChangeLog.md

```diff
@@ -27,6 +27,9 @@
 * Fix email addresses validation in multiple Sql Vulnerability Assessment cmdlets (Update-AzSqlDatabaseVulnerabilityAssessmentSetting, Update-AzSqlServerVulnerabilityAssessmentSetting, Update-AzSqlInstanceDatabaseVulnerabilityAssessmentSetting and Update-AzSqlInstanceVulnerabilityAssessmentSetting).
 * Add support for setting Active Directory Administrator on Managed Instance
 
+## Version 1.14.2
```

Comment:
I think this should be under `## Upcoming Release`

Reply:
Fixed